### PR TITLE
Kakute update

### PR DESCRIPTION
**Pull request by HThuren**
more wiki links for ex. KakuteH7 and KakuteH7v2

### CODE REVIEW

**Review by blckmn — APPROVED** [review]

**Comment by HThuren** [comment]
I have fixed ...KAKUTEH7.config

**Comment by haslinghuis** [comment]
@HThuren yes but as long github complains about conflicting files we can't do anything (but solving the conflict :) )

**Comment by HThuren** [comment]
> @HThuren yes but as long github complains about conflicting files we can't do anything (but solving the conflict :) )

Anything I can do ?

**Review by haslinghuis — APPROVED** [review]
Resolved the conflict.

File: configs/default/HBRO-KAKUTEF7.config [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # Betaflight / STM32F745 (S745) 4.0.0 Apr  3 2019 / 14:32:23 (22b9f3453) MSP API: 1.41
+#wiki https://github.com/betaflight/betaflight/wiki/Board---KAKUTEF7
```
Comment (SteveCEvans) [inline configs/default/HBRO-KAKUTEF7.config:2]: It’s great to have such links, but I think an update is called for. You still refer to 32K loop rate which we no longer support.

“F7 processor allows for faster looptime in theory without overclocking like we do with F4 running 32KHz”

Comment (SteveCEvans) [reply configs/default/HBRO-KAKUTEF7.config:2]: This is outside the scope of this config file update however.

**Review by SteveCEvans — APPROVED** [review]

File: configs/default/HBRO-KAKUTEF7.config (hunk 1 above)
Comment (haslinghuis) [reply configs/default/HBRO-KAKUTEF7.config:2]: Yes documentation lacks updates.

Comment (HThuren) [reply configs/default/HBRO-KAKUTEF7.config:2]: Well, I just made the possibility to refer to the documentation :-), not intented to update the documentation as well. :-)
But..., Holybro still sell the F7, so I will make an update of the documentation. I can also check  F4, also still in sale. :-)

**Comment by HThuren** [comment]
Checkout https://github.com/betaflight/wiki/pull/2, where F7 doc now are updated (and H7 added)

**Pull request merged**